packages/headless-react/typedoc/lib/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {cpSync} from 'node:fs';
+import {dirname, resolve} from 'node:path';
+import {fileURLToPath} from 'node:url';
+// following docs https://typedoc.org/guides/development/#plugins
+// eslint-disable-next-line n/no-unpublished-import
+import {Application, JSX, RendererEvent, Converter} from 'typedoc';
+import {insertAtomicSearchBox} from './insertAtomicSearchBox.js';
```

Comment:
Do you guys think it would be important to create a seperate package that contains the common Coveo css and javascript for typedoc sites ? We could also just duplicate it like I did here.

Reply:
This would include everything in this /lib folder.

Reply:
I can imagine that we'll need to do maintenance on the plugins in the future, which would be duplicated.

Not a big deal, but for instance I'll need to index the typedoc site so its pages show up in search results, and for that I'll probably use a plugin that adds metadata. I can always just make the same modifications twice though.